task/pylint/0.1/pylint.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pylint
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: python, pylint
+    tekton.dev/displayName: pylint
+
+spec:
+  description: >-
+    This task will run pylint on the provided input.
+
+  workspaces:
+    - name: source
+  params:
+    - name: PYTHON
+      description: The used Python version, more precisely the tag for the Python image
+      type: string
+      default: "3.6"
```

Comment:
Why don't we use 3.8 (whichi s the latest) by default?

Reply:
Good point. We could even place there latest as it is the tag for the Python image. Do you think this would make sense or is too confusing having a Python version latest?